src/scenes/Home/components/Weather/WeatherExpanded.jsx

```diff
@@ -1,63 +1,109 @@
 import React from 'react';
 import './styles.scss';
 import { weatherIcon } from './WeatherIcon';
-
+import { weatherForecastIcon } from './WeatherIcon';
 
 const WeatherExpanded = (props) => {
+
+	let firstDayMinTemp = props.weatherForecast.list[0].main.temp_min;
+	let secondDayMinTemp = props.weatherForecast.list[8].main.temp_min;
+	let thirdDayMinTemp = props.weatherForecast.list[16].main.temp_min;
+	let fourthDayMinTemp = props.weatherForecast.list[24].main.temp_min;
+	let fifthDayMinTemp = props.weatherForecast.list[32].main.temp_min;
+
+	let firstDayMaxTemp = props.weatherForecast.list[0].main.temp_max;
+	let secondDayMaxTemp = props.weatherForecast.list[8].main.temp_max;
+	let thirdDayMaxTemp = props.weatherForecast.list[16].main.temp_max;
+	let fourthDayMaxTemp = props.weatherForecast.list[24].main.temp_max;
+	let fifthDayMaxTemp = props.weatherForecast.list[32].main.temp_max;
+
+	//let firstWeatherForecastID = props.weatherForecast.list[0].weather[0].id;
+	//Variable passed into weatherForecastIcon function in JSX - function could be rewritten
+		
+	let firstWeatherForecastDescription = props.weatherForecast.list[0].weather[0].description;
+	let secondWeatherForecastDescription = props.weatherForecast.list[8].weather[0].description;
+	let thirdWeatherForecastDescription = props.weatherForecast.list[16].weather[0].description;
+	let fourthWeatherForecastDescription = props.weatherForecast.list[24].weather[0].description;
+	let fifthWeatherForecastDescription = props.weatherForecast.list[32].weather[0].description;
+
+	//To get day of the week from UNIX time stamp - function could be rewritten
+	
+	let firstDateString = props.weatherForecast.list[0].dt;
+	let firstDate = new Date(firstDateString*1000);
```

Comment:
Can't this be made a lot simpler :) with a for cycle or something?

Reply:
done